tests/python/unittest/test_loss.py

```diff
@@ -81,8 +81,10 @@ def test_ce_loss():
             eval_metric=mx.metric.Loss(), optimizer='adam')
     assert mod.score(data_iter, eval_metric=mx.metric.Loss())[0][1] < 0.05
 
-
-@with_seed(1234)
+# @lanking520: getting rid of the fixed seed
+#               and add atol and rtol
```

Comment:
nit: you should be able to fit this line together with the line above in a single line.

Reply:
Feel free to remove those comments and simply link the issue in PR description.

Reply:
:<